### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amynicol1985**

### CODE REVIEW

**Comment by amynicol1985** [comment]
@TQuock here is the PR for the update based on discussions today.

**Review by jasonblais — DISMISSED** [review]

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 1]
```diff
@@ -248,6 +248,8 @@ Team events and meetups should be planned in advance with ample notice and appro
 
 The [Airfare](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#airfare-and-traveling), [Meals and Entertainment](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#meals-and-entertainment), and [Hotel](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#hotel-and-lodging) policies take precedence even in the case of team events or meetups.
 
+Team off sites where travel and overnight stays are required, have a budget of $250 per person per day, which includes lodging, meals and meeting spaces. It does not include travel costs or external partner and customer dinners.
```
Comment (cwarnermm) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md:251]:
```suggestion
Team off-sites, where travel and overnight stays are required, have a budget of $250 per person per day, which includes lodging, meals, and meeting spaces. It does not include travel costs or external partner and customer dinners.
```

**Review by cwarnermm — DISMISSED** [review]
Non-blocking suggestions provided inline

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md (hunk 1 above)
Comment (TQuock) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
Team off sites where travel and overnight stays are necessary, we encourage staff to use their best judgment. Historically, the average cost is $250 USD per person per day which includes meals, lodging, and meeting spaces. Travel costs like airfare and rideshares or external partner and customer dinners are considered an additional expenditure and should seek [manager approval](https://handbook.mattermost.com/operations/finance/staff-member-expenses/how-to-spend-company-money#manager-approval) first. 
```

Comment (cwarnermm) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: Great update, @TQuock!

Comment (TQuock) [reply company/how-to-guides-for-staff/how-to-spend-company-money/README.md]: @amynicol1985 does the above work for you?

**Review by jasonblais — DISMISSED** [review]

File: company/how-to-guides-for-staff/how-to-spend-company-money/README.md [hunk 2]
```diff
@@ -248,6 +248,8 @@ Team events and meetups should be planned in advance with ample notice and appro
 
 The [Airfare](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#airfare-and-traveling), [Meals and Entertainment](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#meals-and-entertainment), and [Hotel](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#hotel-and-lodging) policies take precedence even in the case of team events or meetups.
 
+Team off sites where travel and overnight stays are necessary, we encourage staff to use their best judgment. Historically, the average cost is $250 USD per person per day which includes meals, lodging, and meeting spaces. Travel costs like airfare and rideshares or external partner and customer dinners are considered an additional expenditure and should seek [manager approval](https://handbook.mattermost.com/operations/finance/staff-member-expenses/how-to-spend-company-money#manager-approval) first.
```
Comment (cwarnermm) [inline company/how-to-guides-for-staff/how-to-spend-company-money/README.md]:
```suggestion
For team off sites, where travel and overnight stays are necessary, we encourage staff to use their best judgment. Historically, the average cost is $250 USD per person per day which includes meals, lodging, and meeting spaces. Travel costs like airfare, ride shares, or external partner and customer dinners are considered an additional expenditure and should seek [manager approval](https://handbook.mattermost.com/operations/finance/staff-member-expenses/how-to-spend-company-money#manager-approval) first. 
```

**Review by cwarnermm — DISMISSED** [review]
Minor, non-blocking feedback provided inline

**Review by jasonblais — APPROVED** [review]

**Pull request merged**